src/webgl/shaders/lighting.glsl

```diff
@@ -73,9 +74,18 @@ LightResult _light(vec3 viewDirection, vec3 normal, vec3 lightVector) {
 
   //compute our diffuse & specular terms
   LightResult lr;
+  float invertValue = (1.0 - (metallic / 100.0));
+  float specularIntensity = mix(0.5 , 1.0, invertValue);
+  float diffuseIntensity = mix(0.2, 1.0, invertValue);
```

Comment:
As a test, I used `fill('red')` and `specularMaterial(100)` on a torus.

With `imageLight()`, when you bring `metalness` up to the maximum value, there's no remnant of the fill color. 

<img width="398" alt="image" src="https://github.com/processing/p5.js/assets/5315059/2b876ea0-7312-40db-9084-e8ca153ad495">

When using `lights()` instead, it's still very prominent:

<img width="395" alt="image" src="https://github.com/processing/p5.js/assets/5315059/c0245f2b-d9f0-4e44-a1d8-f9b843002066">

I think to add metalness as a feature, we should figure out a way to deal with these two cases more consistently. There's two bits involved I think, the color, and the balance of diffuse/specular components.

For color, Threejs had this to say [when metalness was still part of their Phong material:](https://github.com/mrdoob/three.js/commit/bccaa60fa9152ad734e2f53dba45585299e8158d)

>  If set to true the shader multiplies the specular highlight by the underlying color of the object, making it appear to be more metal-like and darker. If set to false the specular highlight is added ontop of the underlying colors.

Maybe that's what we should do too to have consistent color?